docs/source/getting-started/index.rst

```diff
@@ -0,0 +1,82 @@
+.. _getting-started:
+
+Getting Started
+===============
+
+This page will show how to start using MythX in the shortest number of steps.
+
+Quickstart
+----------
+
+The quickest way to see MythX in action is by using a MythX tool such as the :ref:`MythX for Truffle <tooling.truffle>` plugin.
```

Comment:
Maybe we could also mention Remix here? It required even a smaller setup than truffle, right?

Reply:
Agreed, but we need a page about Remix to link to first. I just ticketed that (not sure if it was already ticketed somewhere).

Reply:
I added a link to the Remix extension Github repo on the landing page: https://staging.mythx.io (see the Remix box). However that guide is not very polished, we should maybe make a new one, like a Medium article, and then point to that one.